web-marketplace/src/lib/queries/react-query/registry-server/graphql/getAllProjectsQuery/getAllProjectsQuery.ts

```diff
@@ -15,6 +16,15 @@ export const getAllProjectsQuery = ({
       query: AllProjectsDocument,
     });
 
+    await Promise.all(
```

Comment:
this is another part of the fix for displaying off chain project image/size
our carbonplus grasslands off chain projects metadata are not necessarily compacted